SpatialGDK/Extras/schema/core_types.schema

```diff
@@ -1,9 +1,14 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 package unreal;
 
+type LoadInfo {
```

Comment:
Future devs will be grateful for comments explaining why this exists and what its fields do :)